server/routes/users.js

```diff
@@ -4,17 +4,26 @@ const router = express.Router()
 const { User } = require( '../database' )
 
 router.post( '/sign-up', ( request, response ) => {
+  //if request.session.user, don't even let someone see this page
   const { username, email, password, lat, long } = request.body
 
   User.signUp({ username, email, password, lat, long })
```

Comment:
I would pass in an object instead of individual variables.

Reply:
merci